source/Calamari.Tests/Fixtures/Integration/Process/CommandLineRunnerFixture.cs

```diff
@@ -18,5 +20,24 @@ public void ScriptShouldFailIfExecutableDoesNotExist()
             result.HasErrors.Should().BeTrue();
             output.Errors.Should().Contain(CommandLineRunner.ConstructWin32ExceptionMessage(executable));
         }
+
+        [Test]
+        public void ScriptShouldFailWhenTimeoutIsSpecifiedAfterSomeTime()
```

Comment:
This test would not work on Linux

Reply:
I was thinking along similar lines. I'm not too sure if there are commands both platforms would change, so this test was converted into two instead: one for Windows platforms and one for Linux/Mac. It was done using the TestCategory.CompatibleOS attribute, though I'm only guessing at this point that it is what it is meant for.